google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClientFactory.java

```diff
@@ -220,10 +204,11 @@ public BigtableDataClient createForInstance(
             .build();
     ClientContext clientContext =
         sharedClientContext
+            .getClientContext()
             .toBuilder()
             .setTracerFactory(
                 EnhancedBigtableStub.createBigtableTracerFactory(
-                    settings.getStubSettings(), openTelemetry))
+                    settings.getStubSettings(), sharedClientContext.createOpenTelemetry()))
```

Comment:
this looks wrong, the otel instance in createOpenTelemetry is tied to thje default resource. This probably a carry over bug, but please add a TODO to fix it